src/table/utils/styling-utils.ts

```diff
@@ -27,10 +31,17 @@ export const SELECTION_STYLING = {
   },
 };
 
-// Both index !== -1 and color !== null must be true for the property to be set
-export const isSet = (prop) => prop && JSON.stringify(prop) !== JSON.stringify({ index: -1, color: null });
+/**
+ * Determines if a palette color is set or not. Both index !== -1 and color !== null must be true for it to be unset
+ */
+export const isColorSet = (prop: PaletteColor | undefined): boolean =>
```

Comment:
```suggestion
export const isColorSet = (prop?: PaletteColor): boolean =>
```

Reply:
well, we actually want to enforce that you pass an argument. And sometimes it is undefined and we want to cover that case. But `isColorSet()` does not make sense